lib/admin_profile.xml

```diff
@@ -12,6 +12,10 @@
         <members>*</members>
         <name>BusinessProcess</name>
     </types>
+    <types>
+        <members>*</members>
```

Comment:
@davidmreed - looks like we were missing this in PMM. Hoping for your approval since it's a lib file. Thanks!

Reply:
Nice, elegant solution. Approving.